.travis.yml

```diff
@@ -1,10 +1,12 @@
+sudo: false
+dist: trusty
 language: node_js
 
 node_js:
-  - "6"
+  - "6.10.3"
```

Comment:
Latest stable is `6.11.3`. Should we keep up to date?

Reply:
sure, updated